ode/dialogs/contributor_dialog.py

```diff
@@ -0,0 +1,133 @@
+from PySide6.QtWidgets import QDialog, QVBoxLayout, QHBoxLayout, QPushButton, QWidget, QGridLayout, QLabel, QLineEdit
+from PySide6.QtCore import Qt
+
+
+class ContributorDetailForm(QWidget):
+    """
+    Widget to show the details of a contributor that will be displayed inside a dialog.
+    """
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        layout = QGridLayout()
+
+        self.titleLabel = QLabel()
+        layout.addWidget(self.titleLabel, 0, 0)
+        self.title = QLineEdit()
+        self.title.setMinimumWidth(200)
+        layout.addWidget(self.title, 0, 1)
+
+        self.emailLabel = QLabel()
+        layout.addWidget(self.emailLabel, 1, 0)
+        self.email = QLineEdit()
+        self.email.setMinimumWidth(200)
+        layout.addWidget(self.email, 1, 1)
+
+        self.roleLabel = QLabel()
+        layout.addWidget(self.roleLabel, 0, 2)
+        self.role = QLineEdit()
+        self.role.setMinimumWidth(200)
+        layout.addWidget(self.role, 0, 3)
+
+        self.pathLabel = QLabel()
+        layout.addWidget(self.pathLabel, 1, 2)
+        self.path = QLineEdit()
+        self.path.setMinimumWidth(200)
+        layout.addWidget(self.path, 1, 3)
+
+        layout.setColumnMinimumWidth(1, 150)
+        layout.setColumnMinimumWidth(3, 150)
+        layout.setHorizontalSpacing(20)
+
+        self.setLayout(layout)
+        self.retranslateUI()
+
+    def retranslateUI(self):
+        """
+        Applies the translations to the labels.
+        """
+        self.titleLabel.setText("Title:")
```

Comment:
```suggestion
        self.titleLabel.setText(self.tr("Title:"))
```

Checkout that all the translation strings needs to be wrapped in the `self.tr()` function which is the equivalent of `gettext` in Flask-Babel or the `_` in django.